Dockerfile

```diff
@@ -19,5 +19,6 @@ RUN apk add wget bash
 WORKDIR /bin
 
 COPY --from=builder /go/bin /bin
+COPY /template_migrations /bin/template_migrations
```

Comment:
@ivan-stankov-golang do you think that the migrations could be embed in the binary? I think that most of the times we will be packaging a migration (adding a DB column) together with code changes for a new data point/metric.

Reply:
ok, I changed it to embed